README.md

```diff
@@ -239,6 +239,7 @@ API | Description | Auth | HTTPS | CORS |
 | [LibGen](https://garbage.world/posts/libgen/) | Library Genesis search engine | No | No | Unknown |
 | [Open Library](https://openlibrary.org/developers/api) | Books, book covers and related data | No | Yes | No |
 | [Penguin Publishing](http://www.penguinrandomhouse.biz/webservices/rest/) | Books, book covers and related data | No | Yes | Yes |
+| [PoetryDB](https://github.com/thundercomb/poetrydb#readme) | enables you to get instant data from our vast poetry collection. | No | Yes | Yes |
```

Comment:
- first character of description is not capitalized
- description should not end with .

```suggestion
| [PoetryDB](https://github.com/thundercomb/poetrydb#readme) | Enables you to get instant data from our vast poetry collection | No | Yes | Yes |
```